app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/AOI.scala

```diff
@@ -0,0 +1,48 @@
+package com.azavea.rf.datamodel
+
+import java.sql.Timestamp
+import java.util.{Date, UUID}
+
+import geotrellis.slick.Projected
+import geotrellis.vector.MultiPolygon
+import io.circe.Json
+
+// --- //
+
+/** A Project's Area of Interest.
+  * This represents an area on the map (a `MultiPolygon`) which a user has
+  * set filters for.  If a new Scene entering the system passes these filters,
+  * the Scene will be added to the user's Project in a "pending" state. If the
+  * user then accepts a "pending" Scene, it will be added to their project.
+  */
+case class AOI(
+  /* Database fields */
+  id: UUID,
+  createdAt: Timestamp,
+  modifiedAt: Timestamp,
+  organizationId: UUID,
+  createdBy: String,
+  modifiedBy: String,
+
+  /* Unique fields */
+  area: Projected[MultiPolygon],
+  filters: Json
```

Comment:
Turns out we can't set the `filters` field here to be `CombinedSceneQueryParams`, as that would create a cyclic dependency between `datamodel` and `database`.

The solution is to not worry about it, because the AOI checker tool will depend on both `datamodel` and `database`, and can perform the Json deserialization as it needs.